pyproject.toml

```diff
@@ -14,6 +14,7 @@ dependencies = [
     "requests >= 2.28",
     "rich >= 13.2",
     "sqlfluff >= 2.3.4",
+    "sqlparse >0.4",
```

Comment:
Where is this being used? I must have missed it and GitHub UI is making it annoying to re-open every file.

Reply:
this is being used in the prereq builder. specifically, since we have more than one query per file in some of those cases, this is a nice convenience for splitting those into individual queries for athena queueing rather than writing bespoke logic from scratch.